src/DependencyInjection/Configuration.php

```diff
@@ -32,7 +37,13 @@ public function getConfigTreeBuilder()
 
         $node
             ->children()
-                ->scalarNode('db_driver')->isRequired()->end()
+                ->scalarNode('db_driver')
+                    ->defaultValue('no_driver')
+                    ->validate()
+                        ->ifNotInArray(self::DB_DRIVERS)
+                        ->thenInvalid('SonataMediaBundle - Invalid db driver %s.')
```

Comment:
will it throw the same exception?

Reply:
Before my changes there was `\InvalidArgumentException`, and now it will throw a same exception, see https://github.com/symfony/config/blob/master/Definition/Builder/ExprBuilder.php#L187, but before container compilation started